src/commands/dig.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
         {
             name: 'type',
             description: 'DNS record type to lookup',
-            help: `Supported types:\n${POPULAR_TYPES.map(type => `  ${type}`).join('\n')}\n\nDefaults to \`A\` records.`,
+            help: `Supported types:\n${POPULAR_TYPES.map(type => `  ${type}`)
+                .join('\n')}\n\nDefaults to \`A\` records.`,
```

Comment:
This wrap seems really odd, half way through a string. If the line is too long, might be better to pull `POPULAR_TYPES.map(type => `  ${type}`).join('\n')` into a constant further up